sympy/calculus/singularities.py

```diff
@@ -8,18 +8,23 @@ def singularities(expr, sym):
     """
     Finds singularities for a function.
     Currently supported functions are:
-    - univariate real rational functions
+    - univariate rational(real or complex) functions
```

Comment:
All the tests or docs you have added for only `real-functions` with possible `complex-roots`. You have not added any example of finding singularity of  `complex-function`